server/container/guice/guice-common/src/test/java/org/apache/james/MailsShouldBeWellReceived.java

```diff
@@ -55,6 +71,67 @@
         .await();
 
     ConditionFactory CALMLY_AWAIT_FIVE_MINUTE = CALMLY_AWAIT.timeout(Duration.FIVE_MINUTES);
+    String SENDER = "bob@apache.org";
+    String UNICODE_BODY = "A 8bit encoded body with €uro symbol.";
```

Comment:
really, that's not 8bit, that's UTF-8.

```
$ echo -ne "e" | hexdump -C
00000000  65                                                |e|
00000001
$ echo -ne "€" | hexdump -C
00000000  e2 82 ac                                          |...|
00000003
```
as you can see `e` can be encoded on 8 bits while `€` needs 3 bytes, so 24 bits in UTF-8.

Reply:
Sorry I was copy & pasting, I meant `8bit Content-Transfer-Encoding body with €uro symbol.`